### PR TITLE
Remove deprecated examples using Array of Hashes

### PR DESCRIPTION
**Pull request by jerryaldrichiii**
Due to https://github.com/chef-cookbooks/audit/pull/328 we must update the README.md to remove references to the Array of Hashes syntax.

```ruby
default['audit']['profiles'] = []
default['audit']['profiles'].push(
  name: 'example',
  compliance: 'admin/example'
)
```

becomes

```ruby
default['audit']['profiles'] = {}
default['audit']['profiles']['example'] = {
  compliance: 'admin/example'
}
```

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -85,6 +85,8 @@
 
 # Chef Inspec Compliance profiles to be used for scan of node
 # See README.md for details
+# TODO: Make this `{}` in next major version
+# See: https://github.com/chef-cookbooks/audit/pull/328
 default['audit']['profiles'] = []
```
Comment (Vasu1105) [inline attributes/default.rb]: I am not sure but removing examples will not resolve the main issue of hash of hashes #328  as we still having default as array.

Comment (jerryaldrichiii) [reply attributes/default.rb]: Correct, I was not aware Hash of Hashes was broken. We are going to leave the current docs for now, then when we get Hash of Hashes working we will make sure we include setting the parent to `{}` in all examples.

Comment (teknofire) [reply attributes/default.rb]: we'll also need to update the Automate 2 examples for when this is released as they use the `.push` method as well

Comment (jerryaldrichiii) [reply attributes/default.rb]: @teknofire can you add any links to https://github.com/chef-cookbooks/audit/issues/359

Want to make sure we track them in all the places.

Comment (teknofire) [reply attributes/default.rb]: https://automate.chef.io/docs/data-collection/#setting-up-chef-client-to-send-compliance-scan-data-through-the-chef-server-to-chef-automate

Comment (teknofire) [reply attributes/default.rb]: https://github.com/chef/automate/blob/master/components/automate-chef-io/content/docs/data-collection.md

**Comment by alexpop** [comment]
Merged as part of https://github.com/chef-cookbooks/audit/pull/386

Thanks!

**Pull request closed without merging**